### PR TITLE
Document how to list setfiles

### PR DESCRIPTION
**Pull request by bastelfreak**
extracted this out of https://github.com/voxpupuli/.github/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
 BEAKER_setfile=centos7-64 bundle exec rake beaker
 ```
 
+known working setfiles that we currently (2022-01-06) use:
```
Comment (ekohl) [inline README.md:41]: I'd keep it short
```suggestion
Known working setfiles (2023-01-06) :
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
...
+* ubuntu1604-64
+* ubuntu1804-64
+* ubuntu2004-64
+* ubuntu2204-64
+* debian9-64
+* debian10-64
+* debian11-64
+* centos7-64
+* centos8-64
+* centos9-64
```
Comment (ekohl) [inline README.md:52]: Should we mention that this is really CentOS Stream?

File: README.md [hunk 3]
```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
...
+* almalinux8-64
+* almalinux9-64
+* rocky8-64
+* rocky9-64
```
Comment (ekohl) [inline README.md:56]: I dislike static lists in README files, but wonder what would be better. In the past we had a rake task to list setfiles, but it was based on listing files. Perhaps we should implement something in puppet_metadata to print all the setfiles based on metadata.json.

File: README.md (hunk 2 above)
Comment (bastelfreak) [reply README.md]: good idea. updated it

File: README.md (hunk 3 above)
Comment (bastelfreak) [reply README.md]: yes, I'm also not a big fan of this :(

File: README.md [hunk 4]
```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
...
+* ubuntu1604-64
+* ubuntu1804-64
+* ubuntu2004-64
+* ubuntu2204-64
+* debian9-64
+* debian10-64
+* debian11-64
+* centos7-64
+* centos8-64 (that's actually CentOS stream)
+* centos9-64 (that's CentOS stream as well)
```
Comment (ekohl) [inline README.md:52]: You can use CentOS Linux vs Stream to distinguish between the two
```suggestion
* centos7-64 (CentOS Linux)
* centos8-64 (CentOS Stream)
* centos9-64 (CentOS Stream)
```

File: README.md [hunk 5]
```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
 BEAKER_setfile=centos7-64 bundle exec rake beaker
 ```
 
+Known working setfiles (2023-01-06) :
```
Comment (ekohl) [inline README.md:41]:
```suggestion
Known working setfiles (2023-01-06):
```

File: README.md (hunk 3 above)
Comment (bastelfreak) [reply README.md]: Implemented in https://github.com/voxpupuli/puppet_metadata/pull/142

**Review by TheMeier — APPROVED** [review]

**Pull request merged**